docs/merged_objects.md

```diff
@@ -224,8 +224,106 @@ Some merged objects may have some additional sample metadata columns specific to
+
+
+Metrics for each of the ADTs assayed can be found as a `DataFrame` stored as `rowData` within the alternative experiment:
+
+```r
+rowData(altExp(merged_sce, "adt")) # adt metrics
+```
+
+This data frame contains the following columns with statistics for each ADT:
+
+| Column name | Contents                                                       |
+| ----------- | -------------------------------------------------------------- |
+| `mean-SCPCL000000`      | Mean ADT count across all cells/droplets                       |
+| `detected-SCPCL000000`  | Percent of cells in which the ADT was detected (ADT count > 0 ) |
+| `target_type` | Whether each ADT is a target (`target`), negative/isotype control (`neg_control`), or positive control (`pos_control`). If this information was not provided, all ADTs will have been considered targets and will be labeled as `target` |
+
+Finally, additional metadata for ADT processing can be found in the metadata slot of the alternative experiment
+
+Metadata associated with {ref}`data processing <processing_information:Processing information>` is included in the `metadata` slot as a list.
+Similar to the [parent experiment metadata](#singlecellexperiment-experiment-metadata), the `metadata` will contain three items: `library_id`, `sample_id`, and `library_metadata`.
+The `library_metadata` contains the same list of metadata as its corresponding parent experiment library's metadata and includes one additional field `ambient_profile`, which holds a list of the ambient concentrations of each ADT for the given library.
+
+```r
+metadata(altExp(merged_sce, "adt")) # adt metadata
+```
+
+### Additional SingleCellExperiment components for multiplexed libraries
+
+
+Multiplexed libraries will contain several additional per-cell data columns in the main `colData` data frame (accessed with `colData(merged_sce)` [as above](#singlecellexperiment-cell-metrics)), including cellhash HTO QC statistics and results from demultiplexing analysis.
+Statistics were calculated and demultiplexing was performed separately on each library before objects were merged.
+
+| Column name                 | Contents                                                         |
+| --------------------------- | ---------------------------------------------------------------- |
+| `altexps_cellhash_sum`      | UMI count for cellhash HTOs                                      |
+| `altexps_cellhash_detected` | Number of HTOs detected per cell (HTO count > 0 )                |
+| `altexps_cellhash_percent`  | Percent of `total` UMI count from HTO reads                      |
+| `hashedDrops_sampleid`      | Most likely sample as called by `DropletUtils::hashedDrops`      |
+| `HTODemux_sampleid`         | Most likely sample as called by `Seurat::HTODemux`               |
+| `vireo_sampleid`            | Most likely sample as called by `vireo` (genetic demultiplexing) |
```

Comment:
So I think I like how we present this in the SCE file contents, where we first talk about the HTO stats and then have a separate table where we show the demultiplexing. And we don't ever explicitly say we are doing demultiplexing, we just are adding results from demultiplexing analysis. I don't want people to think we are separating out samples when that's not the case.

Reply:
I updated this https://github.com/AlexsLemonade/scpca-docs/pull/251/commits/7ea39decc3dd5b7e86fd7535ea31ac1251d6c2de & https://github.com/AlexsLemonade/scpca-docs/pull/251/commits/f48ab32554da3d6e758e0afc0f375ab9a8038525